sdk/azidentity/environment_credential_test.go

```diff
@@ -278,6 +283,7 @@ func TestEnvironmentCredential_InvalidClientSecretLive(t *testing.T) {
 
 func TestEnvironmentCredential_UserPasswordLive(t *testing.T) {
 	vars := map[string]string{
+		//for ADFS change clientID
```

Comment:
The client ID for ADFS had to be changed to a specific client such as windows Powershell.